js/placard.js

```diff
@@ -63,30 +63,31 @@
 		constructor: Placard,
 
 		complete: function (action) {
-			var func = this.options['on' + action[0].toUpperCase() + action.substring(1)];
+			var eventName = action === 'accepted' ? 'accept' : 'cancel';
```

Comment:
Sounds good. It allows `'maybe':'imThinkingAboutIt'` to be added in the future. j/k